source/Calamari.Shared/Calamari.Shared.csproj

```diff
@@ -23,10 +23,7 @@
     <RootNamespace>Calamari</RootNamespace>
     <LangVersion>8</LangVersion>
     <Nullable>enable</Nullable>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-    <TargetFrameworks>net452;netstandard2.1;net40</TargetFrameworks>
+    <TargetFrameworks>net40;net452;netstandard2.1</TargetFrameworks>
```

Comment:
Does the latter `TargetFramework` declaration overwrite this one if the condition is correct?

Reply:
This is just an artifact of manual testing, I'll revert.